app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -185,12 +185,12 @@ $(document).ready(function($) {
 });
 
 var get_total_cost = function() {
-  var num_rows = $('#payout_table tbody').find('tr').length;
+  var num_rows = $('#payout_table').find('tr').length;
```

Comment:
@SaptakS @Anish-Agnihotri so would it make sense to keep `tbody` and set `i = 0` ?  Or meh not worth it  ?

`$('#payout_table')` -> first `tr` would be `thead`'s row 
`$('#payout_table tbody') ` -> first `tr` would be the row we need